pyquil/api/qvm.py

```diff
@@ -27,15 +28,20 @@
 class QVMConnection(object):
     """
     Represents a connection to the QVM.
+
+    :ivar NoiseModel noise_model: NoiseModel for the QVM, if received from an input Device instance.
```

Comment:
I don't think we have ivar docs anywhere else in PyQuil

Reply:
We do in e.g. `device.py` and `noise.py`. Do we want them?

Reply:
We don't need to use `:ivar` if we have an explicit constructor. You can add this documentation to where you are setting the instance attribute in the constructor.